src/collectors/pe/decoder.cpp

```diff
@@ -257,7 +258,7 @@ void pe_decoder<_Bits>::get_imports(boost::filesystem::path const &module_path,
         else{
             // not a redirection scheme and no import table...
             // Note: do NOT throw, we might have modules that do not export anything!
-            logging::log(logging::error) << "module " << module_path << " has no import table." << std::endl;
+            logging::log(logging::error) << "ERROR : module " << module_path << " has no import table." << std::endl;
```

Comment:
if you really want to change the dispaly of errors, then it should be done at the logger level, not at the call site.